packages/flutter/lib/src/material/progress_indicator.dart

```diff
@@ -352,6 +354,13 @@ class _CircularProgressIndicatorPainter extends CustomPainter {
       ..color = valueColor
       ..strokeWidth = strokeWidth
       ..style = PaintingStyle.stroke;
+    if (backgroundColor != null) {
+      final Paint backgroundPaint = Paint()
+        ..color = backgroundColor
+        ..strokeWidth = strokeWidth
+        ..style = PaintingStyle.stroke;
+      canvas.drawArc(Offset.zero & size, 0, _twoPi, false, backgroundPaint);
```

Comment:
There's an old comment in the code:
```
  // Canvas.drawArc(r, 0, 2*PI) doesn't draw anything, so just get close.
  static const double _sweep = _twoPi - _epsilon;
```
I assume that you've visually verified that the new 0,2pi code renders correctly. However it might be carry on using _sweep instead of _twoPi, in case Flutter lands on implementations that have the aforementioned problem.

Reply:
Yes it does render 0,2pi as a circle. I had intended to write a note on this in my clarifications to bring this to attention but it seems I forgot. Good eye.